tests/moment.parseFormat-test.js

```diff
@@ -57,3 +57,22 @@ test('GitHub issues', function (t) {
 
   t.end()
 })
+
+test('GitHub issues - special cases', function (t) {
+  // https://github.com/gr2m/moment-parseformat/issues/17
+  t.equal(moment.parseFormat('3/15'), 'M/YY', 'last number is > 12 - 3/15  → M/Y')
+  t.equal(moment.parseFormat('03/15'), 'MM/YY', 'last number is > 12 - 03/15  → MM/Y')
+  t.equal(moment.parseFormat('11/15'), 'MM/YY', 'last number is > 12 - 11/15  → MM/Y')
+
+  t.equal(moment.parseFormat('15/3'), 'DD/M', 'first number is > 12 - 15/3  → DD/M')
+  t.equal(moment.parseFormat('15/15'), 'undefined', 'both numbers are > 12 - 15/15  → undefined')
+  t.equal(moment.parseFormat('03/12'), 'DD/MM', 'both numbers are < 13 - 03/12  → DD/MM')
+  t.equal(moment.parseFormat('3/3'), 'D/M', 'both numbers are < 13 - 3/3  → D/M')
+  t.equal(moment.parseFormat('13/2'), 'DD/M', 'first number is > 12 && <= 31 - 13/2  → DD/M')
+  t.equal(moment.parseFormat('03/12'), 'DD/MM', 'both numbers are < 13 - 03/12 → DD/MM')
+  t.equal(moment.parseFormat('03/03'), 'DD/M', 'both numbers are < 13 - 03/03  → DD/M')
+  t.equal(moment.parseFormat('13/02'), 'DD/M', 'first number is > 12 && <= 31 - 13/02  → DD/M')
+  t.equal(moment.parseFormat('3/12'), 'D/MM', 'both numbers are < 13 - 3/12  → D/MM')
+
+  t.end()
```

Comment:
just a minor: can you remove double spaces before `→` in some of the tests above?